src/main/java/uk/gov/hmcts/cmc/claimstore/models/offers/Settlement.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.cmc.claimstore.models.offers;
+
+import java.util.List;
+
+public class Settlement {
```

Comment:
Decided to change the model suggested on Friday a bit. What I didn't like about it was that it introduced mutable objects within a JSON structure and added unnecessary complexity. What I'm doing now is treating a case settlement as a series of statements made by claimant or defendant. Each statement can be an offer, acceptation of rejection. Business logic will check if given statement makes sense at a given moment, e.g. whether a rejection is submitted before an offer exists, or multiple offers being submitted one after another.